strax/chunk.py

```diff
@@ -277,6 +277,12 @@ def concatenate(cls, chunks):
                 )
             prev_end = c.end
 
+        # try:
```

Comment:
Should we remove the debug comments?

Reply:
Yes, sorry I forgot it. Removed at https://github.com/AxFoundation/strax/pull/838/commits/467205547c18fa235be7d54dae50748b31247349